src/parser/parser_support.c

```diff
@@ -2946,6 +2946,54 @@ pt_has_aggregate (PARSER_CONTEXT * parser, PT_NODE * node)
   return false;
 }
 
+/*
+ * pt_has_define_vars () - check if a statement uses define vars ':='
+ * return	: true if the statement uses define vars ':='
+ * parser (in)	: parser context
+ * stmt (in)	: statement
+ */
+bool
+pt_has_define_vars (PARSER_CONTEXT * parser, PT_NODE * stmt)
+{
+  bool has_define_vars = false;
+
+  parser_walk_tree (parser, stmt, pt_is_define_vars, &has_define_vars, NULL, NULL);
+
+  return has_define_vars;
+}
+
+/*
+ * pt_is_define_vars () - check if a node is a define vars ':='
+ * return : node
+ * parser (in) : parser context
+ * node (in)   : node
+ * arg (in)    :
+ * continue_walk (in) :
+ */
+PT_NODE *
+pt_is_define_vars (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int *continue_walk)
+{
+  bool *is_define_vars = (bool *) arg;
+  *continue_walk = PT_CONTINUE_WALK;
+
+  if (*is_define_vars)
```

Comment:
```suggestion
  if (is_define_vars && *is_define_vars)
```

Reply:
In addition to @hgryoo's comment, how about modifying it as follows?
```

if (is_define_vars == NULL)
  {
     *continue_walk = PT_STOP_WALK;
     return node;
  }
 if(*is_define_vars)
```

Reply:
I'll put an assert.
```
assert (has_define_vars != NULL)
```

Reply:
For pointer variables, I think it is safe to always check the address first and then check the value.
Because assert works only in debug mode, it is difficult to check in release mode.
For this reason, I agree with hgryoo's opinion.

Please consider it as one opinion.

Reply:
I fixed it. thanks.